build.gradle.kts

```diff
@@ -101,7 +101,8 @@ fun Project.configureAndroid() {
                     "IconExpectedSize",
                     "InvalidPackage", // Firestore uses GRPC which makes lint mad
                     "NewerVersionAvailable", "GradleDependency", // For reproducible builds
-                    "SelectableText", "SyntheticAccessor" // We almost never care about this
+                    "SelectableText", "SyntheticAccessor", // We almost never care about this
+                    "InvalidVectorPath" // false positive for fui_ic_anonymous_white_24dp
```

Comment:
I think we may just want to add a single exception for this to the lint baseline rather than ignoring all `InvalidVectorPath` errors since they could be serious in the future.

Reply:
Done.